static-site/src/components/splash/index.jsx

```diff
@@ -65,6 +66,15 @@ class Splash extends React.Component {
           </Styles.CenteredFocusParagraph>
         </FlexCenter>
 
+        <FlexCenter>
+          <Styles.CenteredFocusParagraph theme={{niceFontSize: "20px"}}>
+            <Link to="/ncov-sit-reps">
+              <MdTranslate style={{fontSize: "28px"}}/>
```

Comment:
I'm not familiar with this icon, but happy to have it if it helps things.

Reply:
Sorry, I meant to explicitly mention this!

![Screen Shot 2020-05-13 at 9 17 01 AM](https://user-images.githubusercontent.com/12140437/81838124-8efe8080-94fa-11ea-8e64-2255d331ee13.png)


Adding this icon ^ was my attempt at a compromise solution to this issue brought up by @cassiawag about removing translated sitreps from the splash page:
> Though it will be more difficult to find translated reports if you don't speak English. Because the text to navigate to those reports will be English

I was hoping the icon might encourage non-english speakers to click the link despite it being in english. 

1. I don't know if that will really work 
2. It could be that this is misleading if that icon has traditionally meant "translate this page into current browser settings language" (although if that's the case, linking to the translated sitreps page seems like the next best thing to translating the splash page).
3. Maybe the issue itself isn't make or break as Cassia suggested

So I'm happy to remove it if we don't think it serves as a good solution.

Reply:
It looks like even if not this exact icon, variations of this are commonly used to mean translations. There are some nice other options on Google (With arrows going between the two letters, for example) that might help convey the idea, if we worry about it!

Reply:
Great -- I think it's should stay as is :)